tests/test_rest.py

```diff
@@ -0,0 +1,41 @@
+from invenio_pages import InvenioPagesREST
+from invenio_pages.rest import blueprint
+
+# mock the rest API url prefix
+blueprint.url_prefix = '/api/{}'.format(blueprint.url_prefix)
+
+
+def test_page_content(pages_fixture):
+    """Test page content."""
+    app = pages_fixture
+    app.register_blueprint(blueprint)
+
+    with app.app_context():
+        with app.test_client() as client:
+            resp = client.get('/api/pages/1')
+            assert resp.status_code == 200
+            assert resp.json == {
+                'title': 'Page for Dogs!', 'description': '',
+                'url': '/dogs', 'content': 'Generic dog.',
```

Comment:
Nice! Would it make sense to add a test with some content that includes HTML to be sure that it is correctly rendered/escaped?